transport/server.hh

```diff
@@ -271,6 +271,11 @@ class cql_server::event_notifier : public service::migration_listener,
     std::unordered_map<gms::inet_address, event::status_change::status_type> _last_status_change;
     service::migration_notifier& _mnotifier;
     bool _stopped = false;
+
+    // We want to delay sending NEW_NODE CQL event to clients until the new node
+    // has started listening for CQL requests.
+    // See https://github.com/scylladb/scylla/issues/7301
```

Comment:
I don't think we should put links to issues for stuff that we actually fixed. It's sufficient that it's mentioned in the commitlog.